config-templates/ibm/ibm-vpc-block-csi-driver/5.0/storage-class-parameters.json

```diff
@@ -77,6 +96,10 @@
         "required": "true",
         "default": "false",
         "category": "config",
+        "regex": "^(?:true|false)$",
+        "min-length": 4,
+        "max-length": 5,
+        "supported-values": ["true", "false"],
         "type": "text"
```

Comment:
Check if type should be `boolean`

Reply:
According to docs link provided in the same parameters description, its a boolean param
https://cloud.ibm.com/docs/containers?topic=containers-vpc-block#vpc-block-encryption

Reply:
Agree but here we have to change `type: "text"` ---> `type: "boolean"`

As you see in UI, value is `text` instead of drop down
<img width="425" alt="Screenshot 2023-02-20 at 12 25 35 PM" src="https://user-images.githubusercontent.com/34068620/220034585-baa0e115-138e-4293-8149-034766b0bf72.png">

Reply:
@GunaKKIBM Could you please confirm this?

Reply:
This needs to be of type text. I verified once by giving boolean value, it doesn't work.
```
kubectl create -f ibmc-vpc-block-custom-StorageClass.yaml 
Error from server (BadRequest): error when creating "ibmc-vpc-block-custom-StorageClass.yaml": StorageClass in version "v1" cannot be handled as a StorageClass: json: cannot unmarshal bool into Go struct field StorageClass.parameters of type string
```